examples/abbas_work_area/datastore.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class DataStore(ABC):
+    @abstractmethod
+    def format(self, contact_record):
```

Comment:
Shouldn't data store deal with storing and loading data from a file, DB or and API ?